### PR TITLE
Removed quotes from variable type so it's compatible with Terraform 0.12

### PR DESCRIPTION
**Pull request by tyd**
Fixed the example in the README.yaml and generated README.md

### CODE REVIEW

File: README.yaml [hunk 1]
```diff
@@ -75,7 +75,7 @@ usage: |-
   }
 
   variable "forward_emails" {
-    type = "map"
+    type = map
```
Comment (aknysh) [inline README.yaml]:
```suggestion
    type = map(list(string))
```

**Review by aknysh — CHANGES_REQUESTED** [review]
thanks @tyd 
Please see comments and please rebuild README by executing:

```
make init
make readme/deps
make readme
```

**Comment by tyd** [comment]
@aknysh Thanks, the suggested change has been applied and the README.md generated based on the make instructions.

**Review by aknysh — APPROVED** [review]
thanks @tyd

**Pull request merged**